app/components/button.jsx

```diff
@@ -0,0 +1,187 @@
+import React, {useState, useRef} from 'react';
```

Comment:
please add
```
// https://github.com/EnCiv/civil-pursuit/issues/43
```
to the top of this file and to the top of the story file.  This is something from the style guide that you probably haven't seen but now's the time to start.